internal/civisibility/integrations/gotesting/testing_test.go

```diff
@@ -311,6 +314,8 @@ func TestRetryAlwaysFail(t *testing.T) {
 	t.Fatal("Always fail")
 }
 
+func TestNormalPassingAfterRetryAlwaysFail(t *testing.T) {}
```

Comment:
As I understand, the purpose of this is to have a passing test that runs after the failing ones, right? But is it guaranteed to run in the order they appear in the file?

Reply:
yes, that's the normal go test behaviour, for now.